modules/ztp-querying-the-policy-compliance-status-for-each-cluster.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-querying-the-policy-compliance-status-for-each-cluster_{context}"]
+= Querying the policy compliance status for each cluster
+
+After you have created the validator inform policies for your clusters and pushed them to the
+the zero touch provisioning (ZTP) Git repository you can check the status of each cluster for policy compliance.
+
+.Procedure
+
+. Query the status of the spoke clusters using either the {rh-rhacm-first} web console or the CLI:
++
+* If you want to query status from the {rh-rhacm} web console, click *Governance => Find policies =>
```

Comment:
Minimalism. I don't think you need "If you want to", just "To ..." for the first three steps.

Reply:
change made